packages/legend-application-studio/src/components/editor/StatusBar.tsx

```diff
@@ -99,7 +99,9 @@ export const StatusBar = observer((props: { actionsDisabled: boolean }) => {
 
   // Problems
   const error = editorStore.graphState.error;
-  const warnings = editorStore.graphState.warnings;
+  const warnings = editorStore.graphState.removeDependencyProblems(
```

Comment:
this is redundant, we can do this in `EditorGraphState`

Reply:
true!